TeamsProject/main.py

```diff
@@ -0,0 +1,172 @@
+import selenium
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.chrome.options import Options
+import yaml
+import sys
+import time
+import pyautogui as magic
+from selenium.common.exceptions import NoSuchElementException
+from datetime import date
+from datetime import datetime
+
+
+sys.tracebacklimit = 0
+
+
+settings_path = "D:\Desktop\Python Documents\TeamsProject\settings.yaml"
+with open(settings_path) as f:
+    settings = yaml.load(f, Loader=yaml.FullLoader)
+
+
+logindetails = settings['logindetails']
+username = logindetails['username']
+password = logindetails['password']
+
+mon = settings['monday']
+tue = settings['tuesday']
+wed = settings['wednesday']
+thur = settings['thursday']
+fri = settings['friday']
+
+
+def initiation():
+    print("Class starting..")
+    print("Initiating Bot!")
+    print("Starting Teams")
+    global driver
+    options = webdriver.ChromeOptions()
+    options.add_experimental_option('excludeSwitches', ['enable-logging'])
+    driver = webdriver.Chrome(executable_path="C:\Program Files (x86)\chromedriver.exe", options=options)
+    driver.get("https://teams.microsoft.com")
+    driver.maximize_window()
+    usernameentry()
+    time.sleep(1)
+    passentry()
+    time.sleep(1)
+    findingcal()
+    time.sleep(1)
+    findingactivejoinbutton()
+    time.sleep(1)
+    allow()
+    time.sleep(1)
+    finaljoin()
+    time.sleep(1)

```

Comment:
This would work (not always), it's always a good practice to wait the page to load specific elements instead of waiting 1 second.